packages/ui-tree-browser/src/TreeBrowser/theme.js

```diff
@@ -22,13 +22,28 @@
  * SOFTWARE.
  */
 
-export default function ({ colors, spacing, typography, borders }) {
-  return {
-    controlsTopMargin: spacing.small,
-    borderRadius: borders.radiusMedium,
+/**
+ * Generates the theme object for the component from the theme and provided additional information
+ * @param  {Object} theme The actual theme object.
+ * @param  {Object} themeOverride User provided overrides of the default theme mapping.
+ * @return {Object} The final theme object with the overrides and component variables
+ */
+const generateComponentTheme = (theme, themeOverride = {}) => {
+  const { colors, spacing, borders } = theme
 
-    focusOutlineWidth: borders.widthMedium,
-    focusOutlineColor: colors.borderBrand,
-    focusOutlineStyle: borders.style
+  const componentVariables = {
+    controlsTopMargin: spacing?.small,
```

Comment:
Since you deleted the unused ".controls" style, this is probably not used anywhere either. Can you double-check it please?

Reply:
You were right, it wasn't used elsewhere, I deleted it, thanks.